hollow/src/main/java/com/netflix/hollow/tools/diff/HollowDiffMatcher.java

```diff
@@ -91,7 +95,14 @@ public void calculateMatches() {
         int candidateToMatchOrdinal = toPopulatedOrdinals.nextSetBit(0);
         while(candidateToMatchOrdinal != -1) {
             Object key[] = toIdx.getRecordKey(candidateToMatchOrdinal);
-            int matchedOrdinal = fromIdx.getMatchingOrdinal(key);
+            int matchedOrdinal = -1;
```

Comment:
btw might be good to use `HollowConstants.ORDINAL_NONE` here

Reply:
ack. Kept it -1 like other ordinal checks in this class.